packages/suite-desktop-api/src/api.ts

```diff
@@ -7,6 +7,9 @@ import {
     UpdateInfo,
     UpdateProgress,
     InvokeResult,
+    BootstrapTorEvent,
+    InitialHandShakeClient,
+    HandsakeTorLoad,
```

Comment:
<code>Hands<b>h</b>akeTorLoad</code> (typo) and probably <code>InitialHand<b>s</b>hakeClient</code> to make capital letters consistent.